controller/network/path.go

```diff
@@ -44,6 +45,10 @@ func (self *Path) String() string {
 		out += fmt.Sprintf("->[l/%s]", self.Links[i].Id)
 		out += fmt.Sprintf("->[r/%s]", self.Nodes[i+1].Id)
 	}
```

Comment:
Can we include this in the last router field? I think otherwise it's going to be confusing. So rather than

`[r/lastRouterId] -> [r/localAddress]`  have something like

`[r/lastRouterId (localAddress)]`

Reply:
Updated to append to last router rather than another addition